components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -350,4 +350,14 @@ interface BraveWalletService {
   AddUserAsset(ERCToken token, string chain_id) => (bool success);
   RemoveUserAsset(string contract_address, string chain_id) => (bool success);
   SetUserAssetVisible(string contract_address, string chain_id, bool visible) => (bool success);
+  [Sync] GetDefaultWallet() => (DefaultWallet default_wallet);
```

Comment:
As discussed in slack, let's use async instead.

Reply:
Fixed via interactive rebase, you approved since so just resolving this one.